src/overlay.scss

```diff
@@ -11,6 +11,7 @@ $block: #{$fd-namespace}-overlay;
   $fd-overlay-background-color: rgba(#4a505c, 0.3) !default;
 
   --fd-overlay-background-color: #{$fd-overlay-background-color};
+  $fd-overlay-message-strip-top-margin: fd-space(7);
```

Comment:
none of the `fd` variables should be used

Reply:
This is out of scope of this task. If we want to refactor overlay we should create separate task for this.

Reply:
If we touch it, we fix it

Reply:
If we touch it, we fix it